Dockerfile

```diff
@@ -1,3 +1,9 @@
+# Main Dockerfile used to build production
```

Comment:
If you are changing this, you might need to check [Dockerhub](https://hub.docker.com/r/bigchaindb/bigchaindb/~/settings/automated-builds), if we need to pass the argument `backend` at build time.